dev/com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateDataSourceTest.java

```diff
@@ -81,31 +81,37 @@ public void testAppAuth() throws Exception {
                         .requestProp("X-Validator-User", "dbuser")
                         .requestProp("X-Validator-Password", "dbpass");
         JsonObject json = request.run(JsonObject.class);
-        String err = "Unexpected json response: " + json.toString();
-        assertEquals(err, "DefaultDataSource", json.getString("uid"));
-        assertEquals(err, "DefaultDataSource", json.getString("id"));
-        assertNull(err, json.get("jndiName"));
-        assertTrue(err, json.getBoolean("successful"));
-        assertNull(err, json.get("failure"));
-        assertNotNull(err, json = json.getJsonObject("info"));
-        assertEquals(err, "Apache Derby", json.getString("databaseProductName"));
-        assertTrue(err, json.getString("databaseProductVersion").matches(VERSION_REGEX));
-        assertEquals(err, "Apache Derby Embedded JDBC Driver", json.getString("jdbcDriverName"));
-        assertTrue(err, json.getString("jdbcDriverVersion").matches(VERSION_REGEX));
+        assertSuccessResponse(json, "DefaultDataSource", "DefaultDataSource");
```

Comment:
I don't see why this test code was consolidated elsewhere.  The indirection lessens the readability/debugability of the test case.

Reply:
I disagree here, its still easy to track where the test failed with a stack trace, and having a bunch of redundant code makes the file larger and more cumbersome to navigate. 

If we have 1-3 sets of duplicated code, I usually don't refactor out into helper methods, but when we're doing the same thing 4+ times then a helper method is warranted